scrapy/core/downloader/handlers/http11.py

```diff
@@ -199,9 +208,13 @@ class TunnelingAgent(Agent):
     proxy involved.
     """
 
-    def __init__(self, reactor, proxyConf, contextFactory=None,
-                 connectTimeout=None, bindAddress=None, pool=None):
-        super(TunnelingAgent, self).__init__(reactor, contextFactory, connectTimeout, bindAddress, pool)
+    def __init__(
+        self, reactor, proxyConf, contextFactory=None,
+        connectTimeout=None, bindAddress=None, pool=None
+    ):
+        super(TunnelingAgent, self).__init__(
+            reactor, contextFactory, connectTimeout, bindAddress, pool
+        )
```

Comment:
I don't agree that changes like this one make the code more legible. So if a convention hinders rather than helps code comprehension, is it a good convention to obey?

Reply:
Oh, for me it's quite better. Look the next image, in the second version you can read the full code without needing to scroll horizontally.
![Screenshot_20200228-180238~2](https://user-images.githubusercontent.com/22377678/75569030-a201c880-5a54-11ea-8710-a39a64c30564.png)

I would prefer more width (119 or 120), but for me the readibility, at least in the `super` line, has improved.

Reply:
Okay, but for me, that's _visibility_ rather than readability. With that argument, I could counter that the change didn't help me any, because I'm shortsighted, have a bigger font, and still need to scroll. Or that my device has less screen-width than yours in that screenshot.

Or that since there are now 4 more lines, I can't see the whole function codeblock in my terminal window, so I need to scroll down/up instead of sideways.